src/CMakeLists.txt

```diff
@@ -477,9 +478,15 @@ add_custom_target(clean-std-lib
 add_custom_target(clean-olean
   DEPENDS clean-std-lib)
 
-install(FILES "${CMAKE_SOURCE_DIR}/../bin/leanpkg"
-  DESTINATION bin
-  PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  install(FILES "${CMAKE_SOURCE_DIR}/../bin/leanpkg.bat" "${CMAKE_SOURCE_DIR}/../bin/leanpkg.bat"
```

Comment:
I just cargo culted the "leanemacs.bat" lines below.

I don't use the msys2 shell though, and the ZIP I download from github doesn't have the shell scripts.